src/app/code/community/Omise/Gateway/Model/Payment/Base/Payment.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+
+class Omise_Gateway_Model_Payment_Base_Payment extends Omise_Gateway_Model_Payment {
+    /**
+     * @var string
+     */
+    protected $_code;
+
+    /**
+     * @var string
+     */
+    protected $_formBlockType;
+
+    /**
+     * @var string
+     */
+    protected $_infoBlockType;
+
+    /**
+     * Payment Method features
+     *
+     * @var bool
+     */
+    protected $_isGateway;
+
+    /**
+     * Payment Method features
+     *
+     * @var bool
+     */
+    protected $_canReviewPayment;
+
+    /**
+     * Payment Method features
+     *
+     * @var bool
+     */
+    protected $_isInitializeNeeded;
+
+    /**
+     * @var string
+     */
+    protected $_callbackUrl;
+
+    /**
+     * @param string $code
+     */
+    public function setCode($code)
```

Comment:
Again, since a lot of these don't get changed, we would be better off with class vars or constants, and removing the getters and setters which aren't needed